pkg/server/server.go

```diff
@@ -82,6 +84,10 @@ func New(configuration options.Configuration) (controllerruntime.Manager, error)
 	if err != nil {
 		return nil, err
 	}
+	err = apiextensionsv1.AddToScheme(mgr.GetScheme())
```

Comment:
suggest: Can we put some code comments on why the scheme needs to be added to v1alpha1 as well as apiextensionsv1.

Reply:
added, I do not now why but CRD api is not in core module, but in `apiextensions`, so if I want to get CRD from api server I need to import that module and add to scheme